### ISSUE TITLE
Select: Single select button missing aria-haspopup attribute (4.1.2)

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#955]
### Expected behavior
The element with button role has aria-haspopup attribute set to "dialog" to communicate that the button has an associated popup dialog.

https://www.w3.org/TR/wai-aria-1.1/#aria-haspopup

### Actual behavior
No attribute exists.
